README.md

```diff
@@ -1,5 +1,7 @@
 # Cross-platform normalization enables machine learning model training on microarray and RNA-seq data simultaneously
 
+The full output of a [version](https://github.com/greenelab/RNAseq_titration_results/tree/e73bbc122562d8bf966b4dafa7b96fb4804b85f4) of this analysis is available at Figshare under the DOI: [10.6084/m9.figshare.5035997.v2](https://doi.org/10.6084/m9.figshare.5035997.v2)
```

Comment:
This is the version associated with the paper? Is it the preprint or the submitted version after revision?

This is what [`releases`](https://github.com/greenelab/RNAseq_titration_results/releases) are built to do. It looks like you can [tag an old commit](https://stackoverflow.com/questions/4404172/how-to-tag-an-older-commit-in-git) which will give you the power to specify what the tag corresponds to

Reply:
Figshare DOI LGTM

Reply:
> This is the version associated with the paper? Is it the preprint or the submitted version after revision?

The preprint has not been updated, that corresponds to an older "release" (in quotes because it's not a release yet). The figshare accession is what is associated with the current submission.

Will look into releases, now, thanks!